collectors/audit/nfrr/src/test/resources/collector_items/items.json

```diff
@@ -502,5 +502,43 @@
       "userID": "",
       "url": "https://mygithub.com/Devopscode/NewPrOldCommit"
     }
+  },
+  {
+    "_id" : "5ccc8cd6a63390415925b3a3",
+    "_class" : "com.capitalone.dashboard.model.CollectorItem",
+    "description" : "title test_result audit process",
+    "niceName" : "",
+    "enabled" : true,
+    "errors" : [
+
+    ],
+    "pushed" : true,
+    "collectorId" : "5bb70128039798589fd792ec",
+    "lastUpdated" : 1557101003714,
+    "options" : {
+      "options" : {
+        "jobName" : "test result audit job",
+        "instanceUrl" : "https://jira.company.com/browse/TR-21750"
```

Comment:
Can you update this to follow the collector Item for Audit Result

Reply:
@Sbrenthughes updated